externals/kyuubi-trino-engine/src/main/scala/org/apache/kyuubi/engine/trino/session/TrinoSessionImpl.scala

```diff
@@ -141,7 +144,7 @@ class TrinoSessionImpl(
       require(
         serverScheme.equalsIgnoreCase("https"),
         "Trino engine using username/password requires HTTPS to be enabled")
-      builder.addInterceptor(OkHttpUtil.basicAuth(username, password))
+      builder.addInterceptor(OkHttpUtil.basicAuth(trinoUser, password))
```

Comment:
The ClientSession still uses username(session user), will this cause issues?

Reply:
> The ClientSession still uses `username`(session user), will this cause issues?

It only used for http verification, will have no other impact.

Reply:
as trinoUser is only used for auth, move the definition to here, and rename variable to `user`